.github/CODEOWNERS

```diff
@@ -7,10 +7,10 @@
 * @newrelic/k8s-agents
```

Comment:
Actually, it's not clear that these errors are legitimate -- all of the teams named above (except `php-agent`, which was corrected to `phpc` a few minutes ago) exist. So I'm not sure what the complaint is about.

Reply:
@tippmar-nr do you have admin access to this repo to confirm that the requisite explicit write access has been granted to each of these teams?

The GHE docs give a touch more detail than the "has write access" warning:

> Users and teams must have explicit write access to the repository, even if the team's members already have access.

Reply:
@fallwith No, I'm just a contributor on this repo; I can't see the "Settings" menu at all.

Reply:
[
<img width="921" alt="Screenshot 2024-10-09 at 3 31 33 PM" src="https://github.com/user-attachments/assets/b87e3436-e1f6-482e-8e56-31a6e0c31d5d">
](url)

Reply:
@fallwith @ramkrishankumarN  I can't remember how DOTNET got access in the repo -- it might well have been by using the self-service tool, though I'm not sure. But it looks like Java, Go, Ruby and PHPC all need to be added to match the configuration on the `newrelic-init-container` repo: 
![image](https://github.com/user-attachments/assets/514b9267-5098-47ab-b2ed-f1ddfac32dce)

Reply:
This should be resolved now.